src/cancelContentPlay.js

```diff
@@ -37,6 +37,16 @@ const cancelContentPlay = function(player) {
       player.el_.style.display = 'block';
       placeholder.remove();
     });
+
+    // On iOS 10 on iPhone, playsinline is added when pinching to return from
+    // fullscreen. Detect fullscreen change, remove placeholder and show player.
+    player.on('fullscreenchange', function () {
+      if (player.el_.hasAttribute('playsinline') && placeholder) {
```

Comment:
Are you seeing the black div when returning from fullscreen? This line says if playsinline and the black div is there - hide it. As I understand it, when returning from fullscreen with pinch-to-close, Safari adds playsinline. 

How are you exiting fullscreen? Pinch-to-close, Done or Fullscreen button?